app/models/event.js

```diff
@@ -0,0 +1,34 @@
+var Sequelize = require('sequelize');
+
+var schema = {
+	payload: {
+		type: Sequelize.JSONB,
+		allowNull: false
+	},
+	type: {
+		type: Sequelize.STRING,
+		allowNull: false
+	},
+	dispatchedAt: {
+		type: Sequelize.DATE,
+		allowNull: false,
+		field: 'dispatched_at'
+	},
+};
+
+var options = {
+	underscored: true,
+	tableName: 'event',
+	indexes: [{
+    	name: 'dispatched_at_index',
+    	method: 'BTREE',
+    	fields: ['dispatched_at']
+    }]
+};
+
+module.exports = function(sequelize, DataTypes) {
+	var Event = sequelize.define('Event', schema, options);
+	var Game = sequelize.import(__dirname + '/game');
+	Event.belongsTo(Game);
+	return Event;
+}
```

Comment:
Also, I'm trying to remember what "Message" and "Conversation" were for. Specifically, do we need to manage those things explicitly in the database or is that the sort of thing we would load based on an event chain and keep in memory. (Possibly same for detailed information about game state in the games table)

Ultimately I'm comfortable leaving that all in for now, but it's a discussion worth having.